pootle/apps/pootle_app/assets.py

```diff
@@ -50,6 +54,19 @@
 register('js_editor', js_editor)
 
 
+rel_path = os.path.join('js', 'select2_l10n')
+select2_l10n_dir = os.path.join(settings.WORKING_DIR, 'static', rel_path)
```

Comment:
When we build the packages (for PyPI), the `node_modules` is not present. That is the reason why we rely on this new `select2_l10n` subdirectory within `static/js/` directory.